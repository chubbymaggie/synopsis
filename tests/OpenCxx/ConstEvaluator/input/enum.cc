enum Color { RED = 0x01, GREEN = 0x02, BLUE = 0x04, WHITE = 0x01 | 0x02 | 0x04};
enum Number { ZERO, ONE, TWO, THREE};
enum Flag { NO = 0, YES, OFF = 0, ON, FALSE = OFF, TRUE = ON};
