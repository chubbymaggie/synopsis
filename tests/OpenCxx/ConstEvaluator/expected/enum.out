enumerator : RED, value : 1
enumerator : GREEN, value : 2
enumerator : BLUE, value : 4
enumerator : WHITE, value : 7
enumerator : ZERO, value : 0
enumerator : ONE, value : 1
enumerator : TWO, value : 2
enumerator : THREE, value : 3
enumerator : NO, value : 0
enumerator : YES, value : 1
enumerator : OFF, value : 0
enumerator : ON, value : 1
enumerator : FALSE, value : 0
enumerator : TRUE, value : 1
